jobs/JGLOBAL_PREP_IODA_OBS

```diff
@@ -0,0 +1,36 @@
+#! /usr/bin/env bash
+
+source "${HOMEgfs}/ush/preamble.sh"
+source "${HOMEgfs}/ush/jjob_header.sh" -e "prepiodaobs" -c "base prepiodaobs"
+
+##############################################
+# Set variables used in the script
+##############################################
+
+
+##############################################
+# Begin JOB SPECIFIC work
+##############################################
+# Generate COM variables from templates
+YMD=${PDY} HH=${cyc} generate_com -rx COM_OBS
+
+
+###############################################################
+# Run relevant script
+EXSCRIPT=${BUFR2IODASH:-${HOMEgfs}/ush/ioda/bufr2ioda/run_bufr2ioda.sh}
```

Comment:
I don't see this script in `HOMEgfs`.

Reply:
Good catch.  I set `BUFR2IODASH` in `config.prepiodaobs` and failed to see this error.   `BUFR2IODASH` currently resides in GDASApp `ush/ioda/bufr2ioda`   I added a soft link to g-w `parm` via `link_workflow.sh`.  The path in `JGLOBAL_PREP_IODA_OBS` was changed to `${HOMEgfs}/ush/run_bufr2ioda.sh`.   Done at 7af58ee3.